src/main/java/com/iridium/iridiumskyblock/configs/Configuration.java

```diff
@@ -1,9 +1,15 @@
 package com.iridium.iridiumskyblock.configs;
 
+/**
+ * The main configuration of IridiumSkyblock.
+ * Is persisted automatically on plugin shutdown and reload.
```

Comment:
this isnt correct, it doesnt persist on shutdown because if somone made changes to the config then did /rl it would save the config (overwriting their changes) and this would annoy people

Reply:
Also wrong for Messages & SQL